build/ci.go

```diff
@@ -359,6 +418,16 @@ func doLint(cmdline []string) {
 	linter := downloadLinter(*cachedir)
 	lflags := []string{"run", "--config", ".golangci.yml"}
 	build.MustRunCommandWithOutput(linter, append(lflags, packages...)...)
+
+	protoc, err := downloadProtoc(*cachedir)
```

Comment:
It might be better to turn the generate thing into its own command in ci.go. It's also useful to call this just to regenerate everything.

I'm putting this here because I feel we shouldn't need protoc to run the linters. It's a weird dependency.